jupyter_book/default_config.yml

```diff
@@ -15,8 +15,19 @@ exclude_patterns            : []  # Patterns to skip when building the book. Can
 # Execution settings
 execute:
   execute_notebooks         : auto  # Whether to execute notebooks at build time. Must be one of ("auto", "force", "cache", "off")
-  cache                     : ""  # A path to the jupyter cache that will be used to store execution artifacs. Defaults to `_build/.jupyter_cache/`
-  exclude_patterns          : []  # A list of patterns to *skip* in execution (e.g. a notebook that takes a really long time)
+  cache                     : ""    # A path to the jupyter cache that will be used to store execution artifacs. Defaults to `_build/.jupyter_cache/`
```

Comment:
can we auto-generate this page from the schema at some point? (not this PR)

Reply:
Yep sounds like a good idea, then the shchema would be the "source of truth" for available values and defaults

Reply:
yeah exactly

Reply:
Added to post PR merge TODOs (see initial PR comment)